src/apps/oauth/oauth.c

```diff
@@ -347,7 +347,7 @@ int main(int argc, char **argv)
         mac_key_size=OAUTH_MAC_KEY_SIZE;
       } 
       strncpy(mac_key,mac_key_val,mac_key_size);
-      mac_key[mac_key_size+1]='\0';
```

Comment:
+1 is a bug: `\0` should go into mac_key_size location because the last symbol is at mac_key_size-1 (starting from 0)